doc/source/index.rst.template

```diff
@@ -1,55 +1,31 @@
-:notoc:
-
 .. pandas documentation master file, created by
 
 .. module:: pandas
 
-********************
-pandas documentation
-********************
+*********************************************
+pandas: powerful Python data analysis toolkit
+*********************************************
 
 **Date**: |today| **Version**: |version|
 
 **Download documentation**: `PDF Version <pandas.pdf>`__ | `Zipped HTML <pandas.zip>`__
 
-:mod:`pandas` is an open source, BSD-licensed library providing high-performance,
-easy-to-use data structures and data analysis tools for the `Python <https://www.python.org/>`__
-programming language.
-
-----------------------------------
-How the documentation is organized
-----------------------------------
-
-Pandas has a lot of documentation. A high-level overview of how it’s organized will help you know where to look for certain things:
-
-TODO -> 4 blokken (cfr. coming from)
-
-- :ref:`getting_started` contains a :ref:`_gentle_intro` first introduction to some of the main pandas core concepts. Furthermore, it contains an overview of
-:ref:`community_guides` community provided tutorials.
-- :ref:`user_guide` provides in-depth information on the key concepts of pandas with useful background information and explanation.
-- :ref:`api` contains a technical reference of the pandas API and available functionalities. The reference describes how the methods work and which parameters
-can be used. It assumes that you have an understanding of the key concepts.
-- Saw a typo in the documentation? Want to improve existing functionalities. The :ref:`development` guidelines will guide you through the process of setting up a development environment.
-
-------------
-Getting help
-------------
-
+**Useful links**:
+`Binary Installers <https://pypi.org/project/pandas>`__ |
+`Source Repository <https://github.com/pandas-dev/pandas>`__ |
 `Issues & Ideas <https://github.com/pandas-dev/pandas/issues>`__ |
 `Q&A Support <https://stackoverflow.com/questions/tagged/pandas>`__ |
 `Mailing List <https://groups.google.com/forum/#!forum/pydata>`__
 
------------------------------
-pandas documentation overview
------------------------------
+:mod:`pandas` is an open source, BSD-licensed library providing high-performance,
+easy-to-use data structures and data analysis tools for the `Python <https://www.python.org/>`__
+programming language.
 
-`Binary Installers <https://pypi.org/project/pandas>`__ |
-`Source Repository <https://github.com/pandas-dev/pandas>`__ |
+See the :ref:`overview` for more detail about what's in the library.
 
 {% if single_doc and single_doc.endswith('.rst') -%}
 .. toctree::
     :maxdepth: 3
-    :titlesonly:
```

Comment:
@stijnvanhoey this part might still be needed to get the correct toctree? (so it doesn't use the section headers on the getting_started/index.rst page)

Reply:
Correct! I did a checkout to a wrong version to provide the input for your PR.